schutzbot/deploy.sh

```diff
@@ -51,3 +51,14 @@ fi
 greenprint "Installing the Image Builder packages"
 # Note: installing only -tests to catch missing dependencies
 retry sudo dnf -y install osbuild-composer-tests
+
+# Provision the software under test.
+/usr/libexec/osbuild-composer-test/provision.sh
+
+# RHEL 8.4 still requires the nightly repos to be used
```

Comment:
I think this chunk should be done in `provision.sh`, and `deploy.sh` should be kept generic. The thinking being that `deploy.sh` is specific to running these tests in schutzbot, but they should behave the same if you just execute them from the test RPM on a non-schutzbot system too.

Reply:
So, each test should individually call `provision.sh` if they need it and `deploy.sh` wouldn't be needed for the tests to run unless they are in a schutzbot system? This makes sense to me.